test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -975,5 +1031,152 @@ public void TestSavingNormalizerWithMultidimensionalVectorInput()
 
             Assert.True(dimensions1.SequenceEqual(dimensions2));
         }
+
+        [Fact]
+        public void TestMinHeapForMedianNormalizer()
+        {
```

Comment:
Instead of static values, would it be better to add tests that compute the median with and without the heap and verify that both produce the same results?

Reply:
Yep. This has now been added. The specific tests you highlighted are just testing the heap in general, but I have added another test that tests the median dynamically and compares with the naive LINQ implementation.